lib/govuk_index/page_traffic_loader.rb

```diff
@@ -7,28 +7,30 @@ def initialize(iostream_batch_size: 250)
     end
 
     def load_from(iostream)
-      new_index = index_group.create_index
-      @logger.info "Created index #{new_index.real_name}"
+      Clusters.active.each do |cluster|
+        new_index = index_group(cluster).create_index
+        @logger.info "Created index #{new_index.real_name}"
 
-      old_index = index_group.current_real
-      @logger.info "Old index #{old_index.real_name}"
+        old_index = index_group(cluster).current_real
+        @logger.info "Old index #{old_index.real_name}"
 
-      old_index.with_lock do
-        @logger.info "Indexing to #{new_index.real_name}"
+        old_index.with_lock do
+          @logger.info "Indexing to #{new_index.real_name}"
 
-        in_even_sized_batches(iostream) do |lines|
-          GovukIndex::PageTrafficWorker.perform_async(lines, new_index.real_name)
+          in_even_sized_batches(iostream) do |lines|
+            GovukIndex::PageTrafficWorker.perform_async(lines, new_index.real_name)
```

Comment:
I think the `PageTrafficWorker` needs to know the cluster as well.  See https://github.com/alphagov/search-api/blob/master/lib/govuk_index/page_traffic_worker.rb#L15

Reply:
Thanks, you're right. I've updated the commit.